src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -241,31 +241,48 @@
+                                    </div>
+                                    <div style="display: inline">
+                                        <f:checkbox name="loadProjectListOnStartup"
+                                             checked="${it.config.loadProjectListOnStartup}"
+                                             default="${com.sonyericsson.gerrithudsontrigger.config.Config.DEFAULT_LOAD_PROJECT_LIST_ON_STARTUP}"/>
+                                    </div>
                                 </div>
-                                <div style="display: inline">
-                                    <f:checkbox name="loadProjectListOnStartup"
-                                         checked="${it.config.loadProjectListOnStartup}"
-                                         default="${com.sonyericsson.gerrithudsontrigger.config.Config.DEFAULT_LOAD_PROJECT_LIST_ON_STARTUP}"/>
+                                <div>
+                                    <div style="width: 200px; display: inline-block;">
+                                        ${%Refresh Project List Interval} (seconds)
+                                    </div>
+                                    <div style="display: inline;">
+                                         <f:textbox name="projectListRefreshInterval"
+                                            value="${it.config.projectListRefreshInterval}"
+                                            default="${com.sonyericsson.gerrithudsontrigger.config.Config.DEFAULT_PROJECT_LIST_REFRESH_INTERVAL}"
+                                            checkUrl="'${rootURL}/${serverURL}/projectListRefreshIntervalCheck?value='+escape(this.value)"
+                                            style="max-width: 100px;"/>
+                                    </div>
                                 </div>
-                            </div>
-                            <div>
-                                <div style="width: 200px; display: inline-block;">
-                                    ${%Refresh Project List Interval} (seconds)
-                                </div>
-                                <div style="display: inline;">
-                                     <f:textbox name="projectListRefreshInterval"
-                                        value="${it.config.projectListRefreshInterval}"
-                                        default="${com.sonyericsson.gerrithudsontrigger.config.Config.DEFAULT_PROJECT_LIST_REFRESH_INTERVAL}"
-                                        checkUrl="'${rootURL}/${serverURL}/projectListRefreshIntervalCheck?value='+escape(this.value)"
-                                        style="max-width: 100px;"/>
-                                </div>
-                            </div>
-                        </f:entry>
+                            </f:entry>
+                        </j:if>
+                        <j:if test="${instance.isProjectCreatedEventsSupported()}">
```

Comment:
If you aren't connected I guess you'd need to show some hybrid of both possible options, so it might be easier to do something simpler and always show both in some way.